js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -2864,6 +2864,26 @@ module.exports = {
                     }
                     return -1;
                 },
+
+                /**
+                 * @name GridBaseMethods.getVisibleColumnIndex
+                 * @publicName getVisibleColumnIndex(id)
+                 * @param1 id:number|string
+                 * @return number
+                 */
+                /**
+                 * @name dxTreeListMethods.getVisibleColumnIndex
```

Comment:
GridBaseMethods are base for dxDataGrid and dxTreeList. So the doc comments for dxTreeListMethods.getVisibleColumnIndex can be removed